### PR TITLE
Clean Customers section

### PR DESCRIPTION
**Pull request by JanJoris**
Creating better documentation and conventions.

### CODE REVIEW

File: paths/customers/create.yaml [hunk 1]
```diff
@@ -1,22 +1,24 @@
...
+    For Stievie, the Customer will be created in [Gigya](http://www.gigya.com/).
   consumes:
   - application/json
   produces:
   - application/json
   parameters:
     - name: Customer object
       in: body
-      description: "Contains the customer data"
+      description: "Customer data"
```
Comment (bmoelans) [inline paths/customers/create.yaml]: double qoutes needed?

File: paths/customers/migrate.yaml [hunk 2]
```diff
@@ -1,15 +1,19 @@
 put:
   tags:
     - Customers
-  summary: "Migrate a customer from Gigya to SBSS"
+  summary: Migrate Customer
+  description: >
+    This context is only used for Stievie. When creating a Customer for Stievie, it is only created in Gigya.
+    As have no idea where the logged in Customer was created, we need to apply our data validation rules to it.
```
Comment (bmoelans) [inline paths/customers/migrate.yaml]: As **we** have no idea

File: paths/customers/migrate.yaml [hunk 3]
```diff
@@ -1,15 +1,19 @@
...
+    This endpoint will gather all required data, update Gigya and create a Customer in our BSS solution.
   consumes:
   - application/json
   produces:
   - application/json
   parameters:
     - name: Customer object
       in: body
-      description: "Contains the customer data to be updated"
+      description: "The Customer data to be updated"
```
Comment (bmoelans) [inline paths/customers/migrate.yaml]: double quotes

File: paths/customers/create.yaml (hunk 1 above)
Comment (JanJoris) [reply paths/customers/create.yaml]: Nope, will remove them.

**Pull request merged**